lib/LMSManagers/LMSCustomerManager.php

```diff
@@ -1290,6 +1290,19 @@ public function getCustomerAddresses($id, $hide_deleted = false ) {
             return array();
         }
 
+        $tmp = array_merge(
+            $this->GetCustomerNetDevs($id),
+            $this->GetCustomerNodes($id)
+        );
```

Comment:
@kosaa sprawdzałeś czy to coś powyżej nie wygeneruje czasem ostrzeżeń jak jeden z wyników z arrach_merge() nie będzie array()?